frontend/src/modules/machinery/components/Translation.css

```diff
@@ -44,6 +44,12 @@
     color: #7BC876;
 }
 
+.machinery .translation p {
+    min-height: 22px;
+    text-align: start;
+    white-space: pre-wrap;
+}
```

Comment:
Moving to the right file and properties to keep formatting of multiline strings in line with Translate.Current.